terraso_backend/apps/core/management/commands/loadbackup.py

```diff
@@ -0,0 +1,101 @@
+import json
+import os
+from pathlib import Path
+from tempfile import mkstemp
+
+import structlog
+from django.core import management
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from ._backup_storage import S3BackupStorage
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
+    """Load data from a backup into the database"""
+
+    help = "Load backup JSON files."
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument(
+            "-d",
+            "--directory",
+            type=Path,
+            default=Path("."),
+            help="Directory where backups can be found",
+        )
+        group.add_argument("--s3", action="store_true", help="Retrieve backups from S3 bucket.")
+
+    @staticmethod
+    def _find_latest_backup_dir(directory):
+        """Find the latest backup in a directory.
+
+        Backups are named according to the date they are created.
+        """
+        files = sorted([f for f in directory.glob("backup*") if f.is_file()])
+        if len(files) < 2:
+            raise RuntimeError(f"Couldn't find any backup files in {directory}")
+        # default is to sort ascending, so latest files at end
+        return files[-2:]
+
+    @staticmethod
+    def _copy_s3_file(storage, path, suffix):
+        """Copies the S3 file to a temporary file on local filesystem."""
+        _, tmp_file_path = mkstemp(suffix=suffix)
+        with storage.open(path, "rb") as fp, open(tmp_file_path, "wb") as temp_file:
+            for chunk in fp.chunks():
+                temp_file.write(chunk)
```

Comment:
I used the [django-storages](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html) library here because I saw it used in other parts of the code. This implements the Django storage API, which doesn't provide an API to download files to the local file system. However, poking around a bit, there is a way to use the underlying `boto3` library, which should allow this. I will look into this more.